internal/host/host_test.go

```diff
@@ -815,6 +815,23 @@ func defaultInventory() string {
 	return string(b)
 }
 
+func diskInventory(disks []*models.Disk) string {
+	inventory := models.Inventory{
```

Comment:
Why don't you reuse default configuraiton?
```go
inventory := models.Inventory{
		Interfaces: []*models.Interface{
			{
				Name: "eth0",
				IPV4Addresses: []string{
					"1.2.3.4/24",
				},
			},
		},
	}
```
Can probably be a const, it used in `defaultInventory` as well.